### ISSUE TITLE
Strategy for masking target and protein sequences

### ISSUE THREAD

**Issue by durrantmm** [ltschmitt/RecGen#1]
Hello, thanks for the interesting study. Can you explain what your strategy was for masking protein sequences when training the model? How did you decide which characters to mask? How would the model perform if you masked the entire target sequence at once and kept the protein sequence unmasked? Was this explored in the paper?

**Comment by ltschmitt** [comment]
Hi,
no masking was performed. We fed the full sequence into the fully connected neural networks. As far as I am aware masking of the input is not usually done for VAEs.
Cheers,
Lukas

**Comment by durrantmm** [comment]
Hi Lukas,

I ask because in this repo you provide a file called training_data_masked.csv, and in that file there are asterisks * in both the nucleotide sequences and the protein sequences. What's the meaning of those asterisks?

Best,
Matt

**Comment by ltschmitt** [comment]
I see how you come to that conclusion. This was only supposed to be an example, to show the format of the input, all the stars are in place of mutations, so the data itself is not useful. To get the actual protein sequences you need to go to https://www.ebi.ac.uk/ena/browser/view/PRJEB57361

**Comment by durrantmm** [comment]
Thank you, much appreciated.
